modules/installation-ibm-z-user-infra-machines-iso.adoc

```diff
@@ -65,33 +65,32 @@ rd.znet=qeth,0.0.bdf0,0.0.bdf1,0.0.bdf2,layer2=1,portno=0 zfcp.allow_lun_scan=0
 ----
 
 
-** For installations on FCP-type disks, complete the following tasks:
-... Use `rd.zfcp=<adapter>,<wwpn>,<lun>` to specify the FCP disk where {op-system} is to be installed. For multipathing repeat this step for each additional path. 
-... For multipathing, set the following parameter: `rd.multipath=default`.
-... For multipathing, set the install device as: `coreos.inst.install_dev=/dev/mapper/mpatha`.
-... For single-path installation, set the install device as: `coreos.inst.install_dev=sda`.
+** For single-path installations on FCP-type disks, complete the following tasks:
+
+... Use `rd.zfcp=<adapter>,<wwpn>,<lun>` to specify the FCP disk where {op-system} is to be installed.
+... Set the install device as: `coreos.inst.install_dev=sda`.
 +
 [NOTE]
 ====
-If additional LUNs are configured with NPIV, FCP requires `zfcp.allow_lun_scan=0`.  
+If additional LUNs are configured with NPIV, FCP requires `zfcp.allow_lun_scan=0`.
 ====
 ... Leave all other parameters unchanged.
 +
 [NOTE]
 ====
-Additional post-installation steps are required to fully enable multipathing. See “Completing installation on user-provisioned infrastructure” for more information.  
+Additional post-installation steps are required to enable multipathing. Do not enable multipathing during installation. See "Post-installation machine configuration tasks -> Enabling multipathing with kernel arguments on {op-system}" for more information.
```

Comment:
Multipath on Z is both a day one and day two operation. All paths need to be added to the parm line during installation. The enabling of mulitpath is required after installation, as documented.